content/help/ops/traffic-management/locality-load-balancing/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Locality Load Balancing
+description: Information on how to enable and understand Locality Load Balancing.
+weight: 40
+keywords: [locality,load balancing,priority,prioritized]
+---
+
+A locality defines a geographic location within your mesh using the following triplet:
+
+- Region
+- Zone
+- Sub-zone
+
+The geographic location typically represents a data center. Istio uses
+this information to prioritize load balancing pools to control
+the geographic location where requests are sent.
+
+## Enabling Locality Load Balancing
+
+This feature is experimental and off by default in 1.1. To enable locality load balancing,
+set the `PILOT_ENABLE_LOCALITY_LOAD_BALANCING` environment variable in all Pilot instances.
+
+Currently, the service discovery platform populates the locality automatically.
+In Kubernetes, a pod's locality is determined via the [well-known labels for region and zone](https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#failure-domain-beta-kubernetes-io-region)
+on the node it is deployed. If you are using hosted Kubernetes service your cloud provider
+should configure this for you. If you are running your own Kubernetes cluster you will need
+to add these labels to your nodes. The sub-zone concept doesn't exist in Kubernetes.
+As a result, this field does not need to be configured.
+
+## Locality-Prioritized Load Balancing
+
+_Locality-prioritized load balancing_ is the default behavior for _locality load balancing_ .
+In this mode, Istio tells Envoy to prioritize traffic to the services most closely matching
```

Comment:
This whole description should be talking about traffic to workloads, not services, right?